src/PrestaShopBundle/Translation/PrestaShopTranslatorTrait.php

```diff
@@ -58,20 +58,28 @@ public function trans($id, array $parameters = [], $domain = null, $locale = nul
             $locale = null;
         }
 
-        $translated = parent::trans($id, [], $this->normalizeDomain($domain), $locale);
+        $isSprintfString = $this->isSprintfString($id);
```

Comment:
You should only do this is `$parameters` is not empty, to avoid performing the regex when it's not necessary